Containerfile

```diff
@@ -0,0 +1,34 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+# Copy the files of esi leap inside the container
+
+COPY . .
```

Comment:
Can you add the files that need to be copied to the container in this repo? I think esi-leap.conf should be included and saved under /etc/esi-leap.